source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -125,6 +125,108 @@
         ],
         'returns': 'int32'
     },
+    'ReadDigitalU16': {
```

Comment:
Try to keep these alphabetical so that it will diff better when I generate the metadata (whcih is always alphabetical). Although maybe we'll be able to switch over to generated before submitting.